vm_supervisor/pubsub.py

```diff
@@ -5,13 +5,18 @@
 
 import asyncio
 import logging
+import sys
 from typing import Dict, Hashable, Set
 
 logger = logging.getLogger(__name__)
 
 
 class PubSub:
-    subscribers: Dict[Hashable, Set[asyncio.Queue[set]]]
+    if sys.version_info >= (3, 9):
+        subscribers: Dict[Hashable, Set[asyncio.Queue[set]]]
```

Comment:
Just to be consistent between annotations, stick to either 'Set' or 'set'?

Reply:
Done